CHANGELOG.md

```diff
@@ -40,18 +40,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## v16.0.0
+This release containts the following new modules and updates:
+- Concentrated Liquidity module (x/concentrated-liquidity). Concentrated liquidity is a novel Automated Market Maker (AMM) design introduced by Uniswap that allows for more efficient use of capital. The improvement is
+achieved by providing liquidity in specific price ranges chosen by the user.
+- implement Cosmwasmpool module (x/cosmwasmpool). The CosmWasm Pool Module is an extension for the Osmosis pools, aiming to create a custom module that allows users to create and manage liquidity pools backed by CosmWasm smart contracts. 
+- ProtoRev changes 
+  - Integrate with Concentrated Liquidity Pools. 
+  - Changes dev account payment from once per week to after every trade
+- Dependency Updates 
+  - bump wasmvm to v1.2 and add cosmwasm_1_2 capability
```

Comment:
Could you please add that validators will have to update the wasm cache during the upgrade due to this:

https://osmosis-network.slack.com/archives/C027ELA290B/p1685870833358839?thread_ts=1685716114.561639&cid=C027ELA290B